packages/accounts-password/password_tests.js

```diff
@@ -1697,6 +1697,34 @@ if (Meteor.isServer) (function () {
     }
   )
 
+  Tinytest.add(
+    'passwords - enroll tokens don\'t get cleaned up when reset tokens are cleaned up',
```

Comment:
will fix it. There was another test that I based this on.
`  Tinytest.add(
    'passwords - reset password doesn\t work if email changed after email sent',`
Now I look closely and realize that was a typo. I can fix this as well.